website/assets/js/pages/query-library.page.js

```diff
@@ -6,7 +6,7 @@ parasails.registerPage('query-library', {
     inputTextValue: '',
     inputTimers: {},
     searchString: '', // The user input string to be searched against the query library
-    selectedPurpose: 'all queries', // Initially set to all, the user may select a different option to filter queries by purpose (e.g., "all queries", "information", "detection")
+    selectedKind: 'all queries', // Initially set to all, the user may select a different option to filter queries by purpose (e.g., "all queries", "information", "policies")
```

Comment:
should this comment say "informational" rather than "information"?

Reply:
Good catch, yes it should.